test/parse/bad-string-utf-escape-large.txt

```diff
@@ -0,0 +1,14 @@
+;;; TOOL: wat2wasm
```

Comment:
Should these be called `unicode-escape`?   I'm not actually sure.. I guess the string is utf8 but the thing being escaped is a unicode scalar value right?